dapps/react-dapp-auth/pages/index.tsx

```diff
@@ -1,13 +1,32 @@
 import { Box } from "@chakra-ui/react";
 import AuthClient, { generateNonce } from "@walletconnect/auth-client";
-import { version } from "@walletconnect/auth-client/package.json";
+import { Web3Modal } from "@web3modal/standalone";
 import type { NextPage } from "next";
 import { useCallback, useEffect, useState } from "react";
 import DefaultView from "../views/DefaultView";
-import QrView from "../views/QrView";
 import SignedInView from "../views/SignedInView";
+import type { W3mModal } from "@web3modal/ui";
+import "@web3modal/ui";
 
-console.log(`AuthClient@${version}`);
+declare global {
```

Comment:
Think this ceremony and the explicit `w3m-modal` web component have been factored out in the meantime:

https://docs.walletconnect.com/2.0/web3modal/standalone/installation

Will defer to @0xAsimetriq here though on the details